https://issues.redhat.com/browse/ACM-14538 (#7090)
open-cluster-management/rhacm-docs: clusters/discovery/enable_discovery.adoc

oc get managedcluster <name>
----

. To get a description of your cluster ID, use the {rosa} command line interface by running the following command:

+
[source,bash]
----
rosa describe cluster --cluster=<cluster-name> | grep -o '^ID:.*

> ----

Author: @Randy424 can this command be run from the MCE cluster?

Reviewer: I'm not sure if this answers the question completely but the command is not connected to the MCE cluster that will import the discovered cluster. It is just used to get information from a the target ROSA cluster on your Red Hat OpenShift Cluster Manager console. I usually run it from my local machine, no need to authenticate to the MCE cluster, all you need is your Cluster manager sign-in.

Author: It does answer it. I'm not sure, but it seems like this shouldn't be added as a step, nor for what's new. Maybe we can add the link (https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/rosa_cli/rosa-get-started-cli#rosa-get-started-cli), to this topic since cluster ID is mentioned: https://github.com/stolostron/rhacm-docs/blob/2.12_stage/clusters/discovery/discovery_config_ui.adoc#view-discovered-clusters

Author: During meeting with @Randy424 

To get a description of your {rosa} cluster ID,  run the following command from the {rosa} command line interface:

Reviewer 2: Why would this not be a step? If you add that link, I would add it to the prereqs.

oc get managedcluster <name>
----

> . To get a description of your cluster ID, use the {rosa} command line interface by running the following command:

Author: ```suggestion
. To get a description of your {rosa} cluster ID, run the following command from the {rosa} command line interface:
```
